crates/release_plz_core/src/update_checker.rs

```diff
@@ -0,0 +1,28 @@
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub struct ClientResponse {
+    pub tag_name: String,
+}
+
+pub async fn get_latest_version() -> String {
+    let client = reqwest::Client::builder()
+        .user_agent(env!("CARGO_PKG_NAME"))
+        .build()
+        .unwrap();
+    if let Ok(client_response) = client
+        .get("https://api.github.com/repos/MarcoIeni/release-plz/releases/latest")
+        .send()
+        .await
+    {
+        let tag_name = client_response
+            .json::<ClientResponse>()
+            .await
+            .ok()
+            .unwrap()
+            .tag_name;
+        tag_name
+    } else {
+        "".to_string()
```

Comment:
if there is a connection issue, or github is down, an empty string is returned.
Then when you call this method you do `[13..]`, which results in a panic :/

Try running this without an internet connection, the program should crash. This is not what we want.
If the github api is not available, you can log a warning, but the program shouldn't be interrupted.